### PR TITLE
feat(communities): Add muting category methods

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5975
[Status-go PR](https://github.com/status-im/status-go/pull/2736)
### What does the PR do

Add muting category functions

### Affected areas

Communities

### CODE REVIEW

File: ui/app/AppLayouts/Chat/views/CommunityColumnView.qml [hunk 1]
```diff
@@ -234,6 +234,19 @@ Item {
                     categoryItem = obj
                 }
 
+                StatusMenuItem {
+                    //% "Edit Category"
+                    text: categoryItem.muted ? qsTrId("Unmute category") : qsTrId("Mute category")
```
[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunityColumnView.qml:239]: oh yes? thank you)

[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Chat/views/CommunityColumnView.qml:239]: In fact, I've seen both being used throughout the source code, but it should really be just `qsTr()` unless you want to switch to ID-based translations completely (https://doc.qt.io/qt-5/linguist-id-based-i18n.html#internationalizing-with-text-ids)

[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunityColumnView.qml]: yes I don't know why I used it :) fixed

[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Chat/views/CommunityColumnView.qml]: Ouch my bad... so it seems ID based is the way to go here, so the code should look sth like:

```
//% "Unmute category"
text: categoryItem.muted ? qsTrId("unmute-category") 
                         //% "Mute category"
                         : qsTrId("mute-category")
```

[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Chat/views/CommunityColumnView.qml]: More info in https://github.com/status-im/status-desktop/blob/master/scripts/translationScripts/README.md :)

File: ui/app/AppLayouts/Chat/views/CommunityColumnView.qml [hunk 2]
```diff
@@ -234,6 +234,19 @@ Item {
                     categoryItem = obj
                 }
 
+                StatusMenuItem {
+                    //% "Edit Category"
```
[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunityColumnView.qml:238]: yes :) will fix

[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunityColumnView.qml]: fixed

**Review by osmaczko — APPROVED** [review]
LGTM

**Review by anastasiyaig — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**